index.html

```diff
@@ -5,66 +5,180 @@
+                <a
+                  href="https://libraryfoundationhillsboro.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                  >Library Foundation of Hillsboro</a
+                >
+              </li>
+              <li>
+                Participating as a member and volunteer of the
+                <a
+                  href="https://uuccwc.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                  >Unitarian Universalist Community Church of Washington
+                  County</a
+                >
+              </li>
+              <li>
+                Supporting and spreading awareness for fundraising,
+                crowd-funding and mutual aid efforts that benefit trans
+                <a
+                  href="https://transequality.org/issues/resources/understanding-transgender-people-the-basics"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                  >trans</a
+                >
+                and
+                <a
+                  href="https://www.minus18.org.au/articles/what-does-%22queer%22-mean-anyway/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                  >queer</a
+                >
+                communities
+              </li>
+            </ul>
+          </section>
         </section>
-      </section>
-    </main>
+      </div>
+    </div>
+  </main>
+
+  <footer>
+    <div class="grid">
+      <div class="flex">
+        <section>
+          <h2>Let's work together!</h2>
+          <p>
+            Many of my clients, co-workers, and colleagues would describe me as
+            being someone who's fun to work with and passionate about what I do.
+          </p>
+
+          <p>
+            Feel free to check out recommendations and testimonials toward the
+            bottom of my
+            <a
+              href="https://www.linkedin.com/in/adriantoddross/"
+              rel="noopener noreferrer"
+              target="_blank"
+              >LinkedIn profile</a
+            >. I also have references available upon request.
+          </p>
 
-    <footer>
-      <h2>Get in touch</h2>
-      <p>
-        Send me a message at adriantoddross@gmail.com. I can't wait to hear from
-        you!
-      </p>
-    </footer>
-  </header>
+          <p>
+            Send me a message at
+            <a
+              href="mailto:adriantoddross@gmail.com"
+              rel="noopener noreferrer"
+              target="_blank"
+              >adriantoddross@gmail.com</a
+            >
+            or
+            <a
+              href="https://calendly.com/adriantoddross/coffeechat"
+              rel="noopener noreferrer"
+              target="_blank"
+              >book a coffee chat</a
+            >
+            with me. I can't wait to hear from you!
+          </p>
```

Comment:
💭 Everything above the fold and throughout the page is very text-heavy. Maybe we could lighten the amount of reading by adding some simple button links here.